doc/tools/docgen_cmd.py

```diff
@@ -80,6 +80,38 @@ def format_title(text):
     return f'{text}\n{line}\n\n'
 
 
+def generate_invisible_rst_table(parameters):
+    """
+    Generates an invisible reStructuredText (reST) table with two columns:
```

Comment:
Use "Generate" so that we eventually become closer to the D401 imperative pycodestyle convention?

Reply:
function removed, it was not needed

Reply:
Oh, interesting, I thought this was the key towards fixing the CLI API documentation. Anyways, build warnings are still 1 after removing this and I do not have your high-level viewpoint in all the doc generation parts, so I am re-approving and merging. Revisiting the CLI API documentation improvement is left for a separate PR.